src/main/resources/static/eiffel-rabbit-mq-post-receive-hook.soy

```diff
@@ -15,25 +15,43 @@
     {param value: $config['rabbit.url'] /}
     {param descriptionText: 'RabbitMQ server address to connect to.' /}
     {param errorTexts: $errors ? $errors['rabbit.url'] : null /}
-    {param rows: '4' /}
+    {param rows: '1' /}
   {/call}
 
   {call aui.form.textareaField}
-    {param id: 'rabbit.exchange' /}
+    {param id: 'rabbit.exchange.name' /}
     {param labelContent: 'RabbitMQ exchange name' /}
-    {param value: $config['rabbit.exchange'] /}
+    {param value: $config['rabbit.exchange.name'] /}
     {param descriptionText: 'RabbitMQ exchange names to send messages to.' /}
-    {param errorTexts: $errors ? $errors['rabbit.exchange'] : null /}
-    {param rows: '4' /}
+    {param errorTexts: $errors ? $errors['rabbit.exchange.name'] : null /}
+    {param rows: '1' /}
   {/call}
 
   {call aui.form.textareaField}
+    {param id: 'rabbit.exchange.type' /}
+    {param labelContent: 'RabbitMQ exchange type' /}
+    {param value: $config['rabbit.exchange.type'] /}
+    {param descriptionText: 'RabbitMQ exchange type.' /}
+    {param errorTexts: $errors ? $errors['rabbit.exchange.type'] : null /}
+    {param rows: '1' /}
+   {/call}
+
+  {call aui.form.textareaField}
     {param id: 'rabbit.routingkey' /}
     {param labelContent: 'RabbitMQ routing key' /}
     {param value: $config['rabbit.routingkey'] /}
     {param descriptionText: 'RabbitMQ routing key to use when sending messages.' /}
     {param errorTexts: $errors ? $errors['rabbit.routingkey'] : null /}
-    {param rows: '4' /}
+    {param rows: '1' /}
+  {/call}
+
+  {call aui.form.textareaField}
+    {param id: 'rabbit.deliverymode' /}
+    {param labelContent: 'RabbitMQ delivery mode' /}
+    {param value: $config['rabbitmq.deliverymode'] /}
+    {param descriptionText: 'RabbitMQ delivery mode. Non-persistent (1) or persistent (2)' /}
```

Comment:
It is good that you gave a hint to user about what options are available. Please add info about default value